tests/e2e/no_quorum_notify_daemon_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The etcd Authors
```

Comment:
Would suggest to rename file to `no_quorum_ready_test.go` so we can add more than one test for ExperimentalWaitClusterReadyTimeout flag.

Reply:
Done